packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.controller.js

```diff
@@ -0,0 +1,137 @@
+import { DNS_FILENAME, DATE_FORMAT } from './dashboard.constants';
+
+export default class DomainDnsZoneHistoryDashboardController {
+  constructor(
+    $translate,
+    $stateParams,
+    $state,
+    $q,
+    $document,
+    Domain,
+    Alerter,
+    DNSZoneService,
+    goBack,
+    goToDiffViewer,
+  ) {
+    this.$translate = $translate;
+    this.$stateParams = $stateParams;
+    this.$state = $state;
+    this.Domain = Domain;
+    this.Alerter = Alerter;
+    this.DNSZoneService = DNSZoneService;
+    this.$document = $document;
+    this.$q = $q;
+    this.goBack = goBack;
```

Comment:
This initialization is not needed you can directly call this in template.

Reply:
same as https://github.com/ovh/manager/pull/10233#discussion_r1386526989